### PR TITLE
Add pyproject.toml for Custom Node Registry

### PR DESCRIPTION
**Pull request by haohaocreates**
Hey! My name is HaoHao and I'm from [comfy-org](https://comfy.org/)! We would love to have you join the Comfy Registry, a public collection of custom nodes which lets authors publish nodes by version and automate testing against existing workflows. 

Eventually, the registry will be integrated as a backend for the UI-manager where all nodes will go through a security scan. Nodes that pass these checks will have a verification flag (✅) beside their name on the UI-manager. Feel free to read up more on the registry [here](https://docs.comfy.org/registry/overview#introduction)

Action Required:

- [x] Go to the [registry](https://registry.comfy.org). Login and create a publisher id (everything after the `@` sign on your registry profile). 
- [x] Add the publisher id into the pyproject.toml file.
- [x] Merge the separate Github Actions PR, then merge this PR.

If you want to publish the node manually, [install the cli](https://docs.comfy.org/comfy-cli/getting-started#install-cli) by running `pip install comfy-cli`, then run `comfy node publish`

Otherwise, if you have any questions, please message me on discord at haohao_81202 or join our [server](https://discord.com/invite/comfyorg)!

### CODE REVIEW

**Comment by haohaocreates** [comment]
hey @iemesowum, wanted to send a small nudge and ask if you had any questions!

**Comment by haohaocreates** [comment]
@iemesowum, wondering if there's any way we can support you on this PR!

File: pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+version = "1.0.0"
+license = "LICENSE"
+dependencies = ["openunmix[stempeg]"]
+
+[project.urls]
+Repository = "https://github.com/iemesowum/ComfyUI_IsaacNodes"
+#  Used by Comfy Registry https://comfyregistry.org
+
+[tool.comfy]
+PublisherId = ""
```
Comment (iemesowum) [inline pyproject.toml]:
```suggestion
PublisherId = "iemesowum"
```

File: pyproject.toml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+[project]
+name = "comfyui_isaacnodes"
+description = "This extension offers automatic drums extraction from audio files, as well as a few helper nodes to support my audio synchronization AnimateDiff workflows."
+version = "1.0.0"
+license = "LICENSE"
```
Comment (iemesowum) [inline pyproject.toml]: Should this be `COPYING`?

Does the license get imported, or should full license text go here?

Comment (iemesowum) [inline pyproject.toml]:
```suggestion
license = {file = "COPYING"}
```

Like https://github.com/Comfy-Org/comfy-cli/blob/9ea503d175e8f74a7931f824598ab4d929043e59/pyproject.toml#L7?

Comment (haohaocreates) [reply pyproject.toml]: The value of the license field should just be the file path to your license

Comment (haohaocreates) [reply pyproject.toml]: So if you have COPYING as your license file name in your root directory then, the license value should just be “COPYING” or “LICENSE” , etc

**Comment by haohaocreates** [comment]
hey @iemesowum , you were actually right and it was an error on our part. 

The naming system for the "license" field should `license = {file = "path_to_license_file"}`

if the file is LICENSE, it will be license = { file = "LICENSE" }
If the file is license.txt, it will b license = { file = "license.txt" }

File: pyproject.toml (hunk 2 above)
Comment (haohaocreates) [reply pyproject.toml]: so the file path is relative to the pyproject.toml file so if it's in the root directory it'll be the path to wherever your toml file, so yes it be `{file = COPYING}`

**Review by iemesowum — APPROVED** [review]

**Pull request merged**